src/Controllers/MagicLink/AppController.php

```diff
@@ -267,4 +234,41 @@ public function reset_apps( Request $request )
 
         return response( [ 'message' => 'App order updated' ] );
     }//end reset_apps()
+
+    public function get_apps( Request $request ) {
+        // Fetch all apps
+        $user       = get_current_user_id();
+        $apps       = container()->get( Apps::class );
+        $apps_array = $apps->for_user( $user );
+
+        return response( $apps_array );
+    }//end get_apps()
+
+/**
```

Comment:
Your linter seems to be removing the indentation on the first line of comments for some reason. Please make sure the only linter you are using is PHPCS.